lib/sigh/runner.rb

```diff
@@ -137,11 +137,11 @@ def certificate_to_use
       end
 
       if certificates.count > 1 and !Sigh.config[:development]
-        Helper.log.info "Found more than one code signing identity. Choosing the first one. Check out `sigh --help` to see all available options.".yellow
-        Helper.log.info "Available Code Signing Identities for current filters:".green
+        UI.important "Found more than one code signing identity. Choosing the first one. Check out `sigh --help` to see all available options."
+        UI.important "Available Code Signing Identities for current filters:"
         certificates.each do |c|
           str = ["\t- Name:", c.owner_name, "- ID:", c.id + "- Expires", c.expires.strftime("%d/%m/%Y")].join(" ")
-          Helper.log.info str.green
+          UI.message str.green
```

Comment:
It's not really success, it's just a green status, so I thought it's cleaner to use a message as a level, but still have a green text